strawberryfields/apps/qchem/dynamics.py

```diff
@@ -92,51 +92,40 @@
 from strawberryfields.utils import operation
 
 
-def evolution(modes: int):
-    r"""Generates a custom ``sf`` operation for performing the transformation
-    :math:`U(t) = U_l e^{-i\hat{H}t/\hbar} U_l^\dagger` on a given state.
+def TimeEvolution(w: np.ndarray, t: float):
+    r"""An operation for performing the transformation
+    :math:`e^{-i\hat{H}t/\hbar}` on a given state.
 
-    The custom operation returned by this function can be used as part of a Strawberry Fields
-    :class:`~.Program` just like any other operation from the :mod:`~.ops` module. Its arguments
-    are:
-
-    - t (float): time in femtoseconds
-    - Ul (array): normal-to-local transformation matrix :math:`U_l`
-    - w (array): normal mode frequencies :math:`\omega` in units of :math:`\mbox{cm}^{-1}` that
-      compose the Hamiltonian :math:`\hat{H} = \sum_i \hbar \omega_i a_i^\dagger a_i`
+    This operation can be used as part of a Strawberry Fields :class:`~.Program` just like any
+    other operation from the :mod:`~.ops` module.
 
     **Example usage:**
 
     >>> modes = 2
-    >>> transform =  evolution(modes)
     >>> p = sf.Program(modes)
     >>> with p.context as q:
     >>>     sf.ops.Fock(1) | q[0]
-    >>>     sf.ops.Fock(2) | q[1]
-    >>>     transform(t, Ul, w) | q
+    >>>     sf.ops.Interferometer(Ul.T) | q
+    >>>     TimeEvolution(w, t) | q
+    >>>     sf.ops.Interferometer(Ul) | q
 
     Args:
-        modes (int): number of modes
-
-    Returns:
-        an ``sf`` operation for enacting the dynamics transformation
-    Return type:
-        op
+        w (array): normal mode frequencies :math:`\omega` in units of :math:`\mbox{cm}^{-1}` that
+            compose the Hamiltonian :math:`\hat{H} = \sum_i \hbar \omega_i a_i^\dagger a_i`
```

Comment:
Maybe:

"
normal mode frequencies :math:`\omega` in units of :math:`\mbox{cm}^{-1}` entering 
 the Hamiltonian :math:`\hat{H} = \sum_i^N \hbar \omega_i a_i^\dagger a_i` for :math:`N` independent quantum Harmonic oscillators